UnityProject/Assets/Scripts/Core/SecureStuff/URLOpen/SafeURLOpen.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using UnityEngine;
+
+public static class SafeURL
+{
+
+	public static void Open(string URL)
+	{
+		URL = URL.ToLower();
+
+		if (URL.StartsWith($"http://") || URL.StartsWith($"https://"))
```

Comment:
should we prevent `http` requests? It's better if we force everything to use `https`

Reply:
idk, anything that requires https will just redirect it